docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1717,8 +1749,890 @@ Here are the steps:
+
+     ```
+     appDeployments:
+         Application:
+             myapp:
+                 SourcePath: 'wlsdeploy/applications/myapp-v2'
+                 ModuleType: ear
+                 Target: 'cluster-1'
+     ```
+
+     If you would like to review the entire original model before this change, see [Staging model files](#staging-model-files) in the [Initial use case](#initial-use-case) within this sample.
+
+   - Create a new image from our staged model files using WIT.
+
+     > **Note**: If you are using JRF in this sample, substitute `JRF` for each occurrence of `WLS` in the `imagetool` command line below, plus substitute `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4` for the `--fromImage` value.
+
+     At this point, we have staged all of the files needed for image `model-in-image:WLS-v2`, they include:
+
+     - `/tmp/mii-sample/model-images/weblogic-deploy.zip`
+     - `/tmp/mii-sample/model-images/model-in-image__WLS-v2/model.10.yaml`
+     - `/tmp/mii-sample/model-images/model-in-image__WLS-v2/model.10.properties`
+     - `/tmp/mii-sample/model-images/model-in-image__WLS-v2/archive.zip`
+
+     If you don't see the `weblogic-deploy.zip` file, then it means that you missed a step in the prerequisites.
+
+     Now let's use the Image Tool to create an image named `model-in-image:WLS-v2` that's layered on a base WebLogic image. We've already set up this tool during the prerequisite steps at the beginning of this sample.
+
+     Run the following commands to create the model image and verify that it worked:
+
+     ```
+     $ cd /tmp/mii-sample/model-images
+     $ ./imagetool/bin/imagetool.sh update \
+       --tag model-in-image:WLS-v2 \
+       --fromImage container-registry.oracle.com/middleware/weblogic:12.2.1.4 \
+       --wdtModel      ./model-in-image__WLS-v2/model.10.yaml \
+       --wdtVariables  ./model-in-image__WLS-v2/model.10.properties \
+       --wdtArchive    ./model-in-image__WLS-v2/archive.zip \
+       --wdtModelOnly \
+       --wdtDomainType WLS
+     ```
+
+     If you don't see the `imagetool` directory, then it means that you missed a step in the prerequisites.
+
+     This command runs the WebLogic Image Tool in its Model in Image mode, and does the following:
+     - Builds the final Docker image as a layer on the `container-registry.oracle.com/middleware/weblogic:12.2.1.4` base image.
+     - Copies the WDT ZIP file that's referenced in the WIT cache into the image.
+       - Note that we cached WDT in WIT using the keyword `latest` when we set up the cache during the sample prerequisites steps.
+       - This lets WIT implicitly assume its the desired WDT version and removes the need to pass a `-wdtVersion` flag.
```

Comment:
assume its the desired  -> assume it's the desired

Reply:
done